pkg/distro/rhel/rhel10/distro_internal_test.go

```diff
@@ -53,9 +52,9 @@ func TestEC2Partitioning(t *testing.T) {
 					pt, err := it.GetPartitionTable([]blueprint.FilesystemCustomization{}, distro.ImageOptions{}, rng)
 					require.NoError(t, err)
 
-					bootSize, err := pt.GetMountpointSize("/boot")
-					require.NoError(t, err)
-					require.Equal(t, tt.bootSizeMiB*common.MiB, bootSize)
+					// no /boot on rhel10, see COMPOSER-2324
```

Comment:
We may want a test like https://github.com/osbuild/images/pull/818/files#diff-0242f370d939498c4d5e098575c9dcb58e31cbc21e84480f013fb7f5ff101003R401 here too